nepalingo-web/src/hooks/getTajpuriya.tsx

```diff
@@ -0,0 +1,45 @@
+import useSWR from 'swr';
+import { DictionaryProps, DictionaryResponse } from './useDictionary';
+import { parse } from 'papaparse';
+import fs from 'fs';
+interface WordRecord{
+  word: string;
+  translation: string;
+}
+const fetcher = (url: string) => {
+  const csvfilepath = fs.readFile('Tajpuriya grammer - Sheet1.csv','utf8');
```

Comment:
This is the server side fetching logic that I mentioned in our 1-1 yesterday. Use the `fetch` function instead